src/scripts/content/todoist.js

```diff
@@ -39,6 +40,44 @@ togglbutton.render(
   }
 );
 
+// task detail view
+togglbutton.render('.item_detail:not(.toggl)', { observe: true }, elem => {
+  const container = elem.querySelector('.item_actions');
+
+  const descriptionSelector = () =>
+    elem.querySelector('.item_overview_content').firstChild.textContent;
+
+  const projectSelector = () =>
+    elem.querySelector('.item_detail_parent_name').textContent;
+
+  const tagsSelector = () => {
+    const tags = elem.querySelectorAll('a.item_overview_label');
+
+    return [...tags].map(tag => {
```

Comment:
```suggestion
    return [...tags].map(tag => tag.textContent);
```

really minor code-golf

Reply:
updated :pray: